caching-dependencies_5bb91ef12c7d3a04dd5b5d91.md

```diff
@@ -8,6 +8,41 @@ fast network, so even gigabytes may be cached with ease.
 Here's an example of installing and caching `npm` dependencies in one block,
 then reusing them in subsequent blocks:
 
+```yml
+# .semaphore/semaphore.yml
+agent:
+  machine:
+    type: e1-standard-2
+    os_image: ubuntu1804
+blocks:
+  - name: Install dependencies
+    task:
+      jobs:
+        - name: npm install and cache
+          commands:
+            - checkout
+            - cache restore
+            - npm install
+            - cache store
+
+  - name: Tests
+    task:
+      prologue:
+        commands:
+          - checkout
+          - cache restore
+      jobs:
+        - name: Everything
+          commands:
+            - npm test
+```
+
+The example above uses a cache shortcut command - `cache store` and `cache restore`.
+This means that [caching](caching) script
+will try to recognize your project structure, lookup `lock files` and automatically
+store or restore dependencies in to or from default paths. The full cache command
+is shown in the example below:
```

Comment:
Remove the full caching command and instead link to a page that demonstrates "how to cache custom files and directories".